src/game/AI/ScriptDevAI/scripts/kalimdor/dustwallow_marsh.cpp

```diff
@@ -32,11 +32,24 @@ at_nats_landing
 boss_tethyr
 EndContentData */
 
+#include "AI/BaseAI/UnitAI.h"
+#include "AI/ScriptDevAI/ScriptDevAIMgr.h"
+#include "AI/ScriptDevAI/base/CombatAI.h"
 #include "AI/ScriptDevAI/include/sc_common.h"
 #include "AI/ScriptDevAI/base/escort_ai.h"
+#include "AI/ScriptDevAI/include/sc_grid_searchers.h"
+#include "Common.h"
+#include "Entities/Object.h"
+#include "Entities/ObjectGuid.h"
 #include "Entities/TemporarySpawn.h"
+#include "Entities/Unit.h"
+#include "Entities/UpdateFields.h"
+#include "Globals/ObjectMgr.h"
+#include "MotionGenerators/MotionMaster.h"
+#include "Platform/Define.h"
```

Comment:
I am pretty sure Define.h and Common.h are somewhere in the include paths of the other files else stuff like uint32 wouldnt work